man/.dir-locals.el

```diff
@@ -1,3 +1,4 @@
+; SPDX-License-Identifier: MIT
```

Comment:
So... this doesn't doesn't match the patch description. This file should be CC0-1.0.

But there's a bigger problem: we didn't include spdx headers on those files because it looks quite ugly in the man page. I'm not convinced we should start doing that now. Maybe it'd be better to mention this the licensing readme insteadm?

Reply:
uhm but this is not an example? Also I can't see this line being added to the manpages, it's just a comment no?

Reply:
Eh, I made the comment on a different file, but github attaches it here because the pattern matches. This comment does **not** apply to .dir-locals.el and the man/man man/html helpers.